content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -36,25 +35,13 @@ From this page, choose your starting view or action and click on `+ Step` to bui
 
 {{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_video1.mp4" alt="Filtering network map with search" video=true >}}
 
-If you have a starting point in mind, but aren't sure what your users did next, the funnel step editor automatically loads the top most common **views** and **actions** that users typically see and take next. This allows you to build funnels quicker knowing the paths your users are taking in sequence.
+If you have a starting point in mind, but aren't sure what your users did next, the funnel step editor automatically loads the top most common **views** and **actions** that users typically see.
```

Comment:
```suggestion
If you aren't sure what your users did, the funnel step editor automatically loads common **views** and **actions** that you can choose to add as steps.
```
I just noticed that you can _start_ a funnel with a view or action, which made the "what your users did next" thing make a little less sense!